### DISCUSSION TITLE
Is it possible to spoof firefox into another browser completely?

### DISCUSSION THREAD

**Discussion by bm92aWJldnJvcw** [kkapsner/CanvasBlocker#572] [General]
So I compared the browserleaks.com values of other browser for ex. chrome, edge and changed values from firefox about:config for useragent and appversion and few other things + used CanvasBlocker for just sufficient randomization + Chameleon for IP based Timezone. 
then I tested the browser on pixelscan.net but it detects that I'm on firefox. 

I wonder anyone could make it _undetectable_ "with what we have" that is about:config, canvasblocker and chameleon.

**Comment by crssi** [comment]
NO its not possible.

**Reply by bm92aWJldnJvcw** [reply]
Ah okay. The most I could do was to look fresh firefox browser on each run which is good.

**Reply by crssi** [reply]
I guess you would be interested in securing firefox and in that case I would suggest you to visit https://github.com/arkenfox/user.js/wiki

Cheers

**Reply by kkapsner** [reply]
I agree to crssi. There are too many little quirks that differentiate the browsers. And some of them are very deep in the core.
